configuration/configuration_test.go

```diff
@@ -193,6 +195,22 @@ func TestGetKeycloakEndpointUserinfo(t *testing.T) {
 	assert.NotNil(t, configurationData.GetKeycloakEndpointUserinfo())
 }
 
+func TestMultipleConfigurations(t *testing.T) {
+	t.Parallel()
```

Comment:
Thanks for adding `t.Parallel`. This might help reducing the time CI needs for testing.